msteams-platform/messaging-extensions/how-to/add-authentication.md

```diff
@@ -36,7 +36,7 @@ The sequence is as follows:
 1. After the user signs in, you should close your window and send an **authentication code** to the Teams client.
 1. The Teams client then reissues the query to your service, which includes the authentication code passed in Step 5.
 
-Your service should verify that the authentication code received in step 6 matches the one from step 5. This ensures that a malicious user doesn't try to spoof or compromise the sign in flow. This effectively "closes the loop" to finish the secure authentication sequence.
+Your service should verify that the authentication code received in step 6 matches with step 5. It ensures that a malicious user doesn't try to spoof or compromise the sign in flow, which effectively "closes the loop" to finish the secure authentication sequence.
```

Comment:
Don't combine shorter sentences to make longer ones. Let's use shorter sentences to the extent possible, especially as long as it doesn't impact the meaning of the copy.